Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -42,7 +42,7 @@ private static string GetHandlerPathIfAny(string listenerUrl)
             }
 
             var startHostUrl = listenerUrl.Substring(pos + "://".Length);
-            var endPos = startHostUrl.IndexOf('/', StringComparison.Ordinal);
+            var endPos = startHostUrl.IndexOf("/", StringComparison.Ordinal);
```

Comment:
Why change this?

Reply:
IndexOf(char) might also be faster, then the string one.